pkg/operators/operators_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 The Inspektor Gadget authors
```

Comment:
nit:
```suggestion
// Copyright 2022-2023 The Inspektor Gadget authors
```

Sorry that it took so long :)

Reply:
Haha, typical copy paste error :D